### PR TITLE
Delete unused files

### LINKED ISSUE #278: Remove internal not used factor2num()
There is a non-exported and not used function `factor2num()`. Shouldn't it be removed from **hyperSpec**? What do you think @cbeleites?

### LINKED ISSUE #279: Remove internal not used .call.list()
There is a non-exported and not used function `.call.list()`. Shouldn't it be removed from **hyperSpec**? What do you think @cbeleites?

### LINKED ISSUE #280: Remove internal not used .split.dots
There is a non-exported and not used function `.split.dots()`. Shouldn't it be removed from **hyperSpec**? What do you think @cbeleites?

### PR DESCRIPTION
**Pull request by sangttruong**
.factor2num(),  .call.list(), and .split.dots() turn out to be a non-exported function.
Thus, removing them is necessary to reduce complexity.

Resolve #278, Resolve #279, Resolve #280

### CODE REVIEW

**Comment by GegznaV** [comment]
@sangttruong 

1) This PR is a duplicate of #315, so, please, address our comments and solve the original issues in #315 and not here. Did you base your new PR on the `cbeleites/develop` branch?
2) If you want to solve style issues that are non-breaing (add spaces, change = into <-, etc., other changes that are done with the `styler` package) then no extensive discussion is needed. But if changes are breaking, like a new function or argument names, then it is better to open a new issue before creating a PR.

In conclusion, firstly, fix the solution in #315. Then go to other tasks.

I you have any questions or need clarifications, please, do not hesitate to ask.

**Review by GegznaV — APPROVED** [review]
